_posts/2016-10-07-our-jekyll-workflow.md

```diff
@@ -0,0 +1,903 @@
+    browserSync.reload();
+    callback();
+});
+
+// Static Server + watching files.
+// Note: passing anything besides hard-coded literal paths with globs doesn't
+// seem to work with gulp.watch().
+gulp.task('serve', ['build:local'], function() {
+
+    browserSync.init({
+        server: paths.siteDir,
+        ghostMode: false, // Toggle to mirror clicks, reloads etc. (performance)
+        logFileChanges: true,
+        logLevel: 'debug',
+        open: true        // Toggle to automatically open page when starting.
+    });
+
+    // Watch site settings.
+    gulp.watch(['_config.yml'], ['build:jekyll:watch']);
+
+    // Watch .scss files; changes are piped to browserSync.
+    gulp.watch('_assets/styles/**/*.scss', ['build:styles']);
+
+    // Watch .js files.
+    gulp.watch('_assets/js/**/*.js', ['build:scripts:watch']);
+
+    // Watch image files; changes are piped to browserSync.
+    gulp.watch('_assets/img/**/*', ['build:images']);
+
+    // Watch posts.
+    gulp.watch('_posts/**/*.+(md|markdown|MD)', ['build:jekyll:watch']);
+
+    // Watch drafts if --drafts flag was passed.
+    if (module.exports.drafts) {
+        gulp.watch('_drafts/*.+(md|markdown|MD)', ['build:jekyll:watch']);
+    }
+
+    // Watch html and markdown files.
+    gulp.watch(['**/*.+(html|md|markdown|MD)', '!_site/**/*.*'], ['build:jekyll:watch']);
+
+    // Watch RSS feed XML file.
+    gulp.watch('feed.xml', ['build:jekyll:watch']);
+
+    // Watch data files.
+    gulp.watch('_data/**.*+(yml|yaml|csv|json)', ['build:jekyll:watch']);
+
+    // Watch favicon.png.
+    gulp.watch('favicon.png', ['build:jekyll:watch']);
+});
+```
+
+With this, we can run `gulp serve` to fire up our site locally. The initial
+build takes a bit longer than `jekyll serve` but the convenience of
+automatically opening a browser window, automatic refreshing, and new CSS being
+injected is fantastic!
+
+## The Results
+
+After implementing these changes, our mobile PageSpeed score shot up to 92/100,
+with our desktop score at 95/100!
+
+<img src="/assets/img/blog/pagespeed-insights-results.jpg" class="blog-image-xl" alt="95/100 Google PageSpeed Insights score!">
+
+To pat ourselves on the back a little more, I should mention that we were already
+doing great in the user experience department.
+
+<img src="/assets/img/blog/pagespeed-insights-user-experience.jpg" class="blog-image-xl" alt="Passing Google PageSpeed Insights user experience items.">
+
+That's grounds for a little celebration in my opinion!
+
+<img src="/assets/img/blog/liz-lemon-self-five.gif" class="blog-image" style="max-width: 300px;" alt="Tina Fey as Liz Lemon giving herself an awesome high five.">
+
+## Next Steps
+
+There are a couple things I'd like to improve with this process:
+
+- Once an image is optimized via imagemin it shouldn't be processed every time
+`build:images` runs to save time both when building the site for testing and
+deployment and for spinning up the site locally.
```

Comment:
Break this up in two sentences. Maybe something like: "...runs. **_By addressing this, we'll_** save time both when building the site for testing and deployment and for spinning up the site locally."